src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts

```diff
@@ -17,6 +17,7 @@ export class ConfigurationGroupComponent implements AfterViewInit {
   @Input() serviceStatus = false;
   @Input() from: string;
   @Input() sourceName: string;
+  @Input() isFilterConfigOpen: boolean;
```

Comment:
You mean expanded? How expanded and collapsed matter.

Reply:
We need to do some calculation to show tab navigator buttons. In the case of filter, it was not working properly because by default configuration (and tabs) are hidden and get visible when we expand it. So we need to calculate it again when user expand the filter.